autogen.sh

```diff
@@ -31,25 +31,25 @@ for pvd in "${providers[@]}"; do
   # convert the svg to png for azure provider
   if [ "$pvd" = "onprem" ] || [ "$pvd" = "azure" ]; then
     echo "converting the svg to png using inkscape for provider '$pvd'"
-    python -m scripts.resource svg2png "$pvd"
+    python3 -m scripts.resource svg2png "$pvd"
```

Comment:
Diagrams uses Python3 as the default interpreter, Python3 is standard, and Python2 is deprecated. So, since we can assume that `python` is for version 3, it doesn't seem necessary to specify `python3` I think.

Also, diagrams is using `poetry` as a package manager and virtual environment manager. We have already specified to use Python3 as default version: https://github.com/mingrammer/diagrams/blob/master/pyproject.toml#L13